File: src/main/java/com/gangofconnectfour/powerfourservice/repository/AuthDataRepository.java
package com.gangofconnectfour.powerfourservice.repository;

import com.gangofconnectfour.powerfourservice.model.AuthData;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface AuthDataRepository extends JpaRepository<AuthData, Long> {
}

File: src/main/kotlin/com/gangofconnectfour/powerfourservice/repository/AuthDataRepository.kt
package com.gangofconnectfour.powerfourservice.repository

import com.gangofconnectfour.powerfourservice.model.AuthData
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AuthDataRepository : JpaRepository<AuthData, String> {
}